Styles/avatar.css

```diff
@@ -0,0 +1,40 @@
+.avatar{
+    border-radius: 999px;
```

Comment:
suggestion: percentage can be used instead of px.

Reply:
I took it from the figma design file but sure I will change that.